site/streams.md

```diff
@@ -224,6 +224,46 @@ but some have some queue specific behaviour.
  * [Consumer acknowledgements](./confirms.html) (keep [QoS Prefetch Limitations](#global-qos) in mind)
  * Cancellation of consumers
 
+### <a id="single-active-consumer" class="anchor" href="#single-active-consumer">Single Active Consumer</a>
+
+Single active consumer for streams is a feature available in RabbitMQ 3.11 and more.
+It provides _exclusive consumption_ and _consumption continuity_ on a stream.
+When several consumer instances sharing the same stream and name enable single active consumer, only one of these instances will be active at a time and so will receive messages.
+The other instances will be idle.
+
+The single active consumer feature provides 2 benefits:
+
+* Messages are processed in order: there is only one consumer at a time.
+* Consumption continuity is maintained: a consumer from the group will take over if the active one stops or crashes.
+
+A [blog post](https://blog.rabbitmq.com/posts/2022/07/rabbitmq-3-11-feature-preview-single-active-consumer-for-streams/) provides more details on single active consumer for streams.
+
+### <a id="super-streams" class="anchor" href="#super-streams">Super Streams</a>
+
+Super streams are a way to scale out by partitioning a large stream into smaller streams.
+They integrate with [single active consumer](#single-active-consumer) to preserve message order within a partition.
+Super streams are available in RabbitMQ 3.11 and more.
+
```

Comment:
Again same comment as above what is the reference to "and more"?? Delete or explain further - thanks

Reply:
This is related to streams not part of this PR about Oauth 2.0